examples/long_tutorials/handle_drift/plot_handle_drift.py

```diff
@@ -0,0 +1,342 @@
+#    need to place the code within a  ``if __name__ == "__main__":`` block.
+
+
+num_units = 100  # 250 still too many I think!
+duration = 1000
+
+_, raw_recording, _ = generate_drifting_recording(
+    num_units=num_units,
+    duration=duration,
+    generate_sorting_kwargs=dict(firing_rates=(15, 25), refractory_period_ms=4.0),
+    seed=42,
+    generate_displacement_vector_kwargs=dict(motion_list=[
+            dict(
+                drift_mode="zigzag",
+                non_rigid_gradient=None, # 0.1,
+                t_start_drift=int(duration/10),
+                t_end_drift=None,
+                period_s=int(duration/10),
+            ),
+        ],
+    )
+)
+print(raw_recording)
+
+# %%
+# Before performing motion correction, we will **preprocess** the recording
+# with a bandpass filter and a common median reference.
+
+filtered_recording = si.bandpass_filter(raw_recording, freq_min=300.0, freq_max=6000.0)
+preprocessed_recording = si.common_reference(filtered_recording, reference="global", operator="median")
+
+# %%
+#.. warning::
+#    It is better to not whiten the recording before motion estimation, as this
+#    will give a better estimate of the peak locations. Whitening should
+#    be performed after motion correction.
+
+# %%
+# ----------------------------------------
+# Run motion correction with one function!
+# ----------------------------------------
+#
+# Correcting for drift is easy! You just need to run a single function.
+# We will now run motion correction on our recording using the three
+# presets described above - **rigid_fast**, **kilosort_like** and
+# **nonrigid_accurate**.
+#
+# We can run these presents with the ``preset`` argument of
+# :py:func:`~spikeinterface.preprocessing.correct_motion()`. Under the
+# hood, the presets define a set of parameters by set how to run the
+# 4 submodules that make up motion correction (described above).
+print(motion_options_preset["kilosort_like"])
+
+# %%
+# Now, lets run motion correction with our three presets. We will
+# set the ``job_kwargs`` to parallelize the job over a number of CPU cores—motion
+# correction is computationally intensive and will run faster with parallelization.
+
+presets_to_run = ("rigid_fast", "kilosort_like", "nonrigid_accurate")
+
+job_kwargs = dict(n_jobs=40, chunk_duration="1s", progress_bar=True)
+
+results = {preset: {} for preset in presets_to_run}
+for preset in presets_to_run:
+
+    corrected_recording, motion_info = si.correct_motion(
+        preprocessed_recording, preset=preset,  output_motion_info=True, **job_kwargs
+    )
+    results[preset]["motion_info"] = motion_info
+
+# %%
+#.. seealso::
+#   It is often very useful to save ``motion_info`` to
+#   file, so it can be loaded and visualised later. This can be done by setting
+#   the ``folder`` argument of
+#   :py:func:`~spikeinterface.preprocessing.correct_motion()` to a path to write
+#   the motion output to. The ``motion_info`` can be loaded back with
```

Comment:
Maybe we give the file suffix. the motion output to a `.npy` file (I forget the format to be honest) what do you think?

Reply:
I had a quick look at it seems to write a lot of output to a folder, including parameters `.json` and mostly `.npy` files. So it is not as simple as the text originally implied. In this case I updated to `all motion outputs to` to avoid going too specific?